### PR TITLE
Tests updated

### PR DESCRIPTION
**Pull request by kgromadzki**

### CODE REVIEW

File: whylog/tests/tests_log_reader/test_files/002_match_latest/parsers.yaml [hunk 1]
```diff
@@ -1,10 +1,14 @@
 ---
 name: cause
-regex: ^(\d+) root cause
-params:
-    - {type: time, format: '%s'}
+regex_str: ^(\d+) root cause
+primary_key_groups: [1]
+log_type: default
+convertions: {1: date}
```
Comment (konefalg) [inline whylog/tests/tests_log_reader/test_files/002_match_latest/parsers.yaml]: Is it date, or just int?

File: whylog/tests/tests_log_reader/test_files/002_match_latest/rules.yaml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 causes: [cause]
 effect: effect
-constraints:
-    - {type: latest}
+constraints: []
+# unsupported constraint type
+#    - {type: latest}
```
Comment (konefalg) [inline whylog/tests/tests_log_reader/test_files/002_match_latest/rules.yaml]: I think you can left this uncommented. We will create such constraint and will think about checkin it up. We'll probably have to discuss this.

File: whylog/tests/tests_log_reader/test_files/002_match_latest/rules.yaml [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 causes: [cause]
 effect: effect
-constraints:
-    - {type: latest}
+constraints: []
+# unsupported constraint type
```
Comment (ppolewicz) [inline whylog/tests/tests_log_reader/test_files/002_match_latest/rules.yaml:5]: make a ticket or skiptest for this

File: whylog/tests/tests_log_reader/test_files/009_match_negation/rules.yaml [hunk 4]
```diff
@@ -1,4 +1,6 @@
 ---
 causes: [cause]
 effect: effect
-negative: true
+constraints: []
+# unsupported rule type
+#negative: true
```
Comment (ppolewicz) [inline whylog/tests/tests_log_reader/test_files/009_match_negation/rules.yaml:6]: add a skiptest or ticket for it

File: whylog/tests/tests_log_reader/test_files/010_multiple_files/rules.yaml [hunk 5]
```diff
@@ -1,4 +1,5 @@
 ---
 causes: [cause]
 effect: effect
-node: different
+constraints: []
+#node: different
```
Comment (ppolewicz) [inline whylog/tests/tests_log_reader/test_files/010_multiple_files/rules.yaml:5]: I think Ewa already implemented it?

Comment (kgromadzki) [inline whylog/tests/tests_log_reader/test_files/010_multiple_files/rules.yaml:5]: There is nothing common with Ewa stuff. In the past it meant that whylog should search many files. But now whylog by default search in many files.

**Pull request merged**